From 72675eeaebc67857d0f1f62b5daeaa35506cd207 Mon Sep 17 00:00:00 2001
From: Eric Blake <eblake@redhat.com>
Date: Mon, 5 Dec 2011 10:32:41 -0700
Subject: [PATCH 08/13] spec: add dmidecode as prereq

(cherry-pick of upstream e7dfa46 with extra requires for with_systemd
removed, since the patch that adds that hasn't been backported)

https://bugzilla.redhat.com/show_bug.cgi?id=754909 complains that
because libvirt didn't require dmidecode, that the logs are noisy
and virConnectGetSysinfo needlessly fails.  Even 'virt-what' requires
dmidecode, so it's not that onerous of a dependency.  We may be
able to drop this in the future when we move to parsing sysfs data,
but for now, listing the dependency will help matters.

* libvirt.spec.in (Requires): Sort Requires before BuildRequires.
Add dmidecode.
---
 libvirt.spec.in |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/libvirt.spec.in b/libvirt.spec.in
index 5cb87fb..ba950c6 100644
--- a/libvirt.spec.in
+++ b/libvirt.spec.in
@@ -326,6 +326,8 @@ Requires: device-mapper
 %if %{with_cgconfig}
 Requires: libcgroup
 %endif
+# For virConnectGetSysinfo
+Requires: dmidecode
 
 # All build-time requirements
 BuildRequires: python-devel
-- 
1.7.7.3

